package.json

```diff
@@ -31,7 +35,15 @@
         "yup": "^1.2.0"
     },
     "devDependencies": {
-        "@babel/core": "^7.20.0"
+        "@babel/core": "^7.20.0",
+        "@typescript-eslint/eslint-plugin": "^6.1.0",
+        "@typescript-eslint/parser": "^6.1.0",
+        "eslint": "^8.45.0",
+        "eslint-plugin-react": "^7.32.2",
+        "eslint-plugin-unicorn": "^48.0.0"
+    },
+    "eslintConfig": {
+        "extends": "universe/native"
```

Comment:
I think you may want to make this into an array and add `"plugin:unicorn/recommended"` to it as well

Reply:
so that it looks like this:
`"extends": ["universe/native", "plugin:unicorn/recommended"]`